examples/v1alpha3/README.md

```diff
@@ -308,3 +308,47 @@ Acsess to `http://127.0.0.1:8000/katib`
 ## Clean
 Clean up with [destroy.sh](./MinikubeDemo/destroy.sh) script.
 It will stop port-forward process and delete minikube cluster.
+
+# List of current Katib training container images
+
+- Mxnet mnist example, [source](https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/train_mnist.py).
+
+```
+docker.io/kubeflowkatib/mxnet-mnist-example
+```
+
+- Mxnet mnist example with collecting metrics time, [source](https://github.com/kubeflow/katib/blob/master/examples/v1alpha3/mxnet-mnist-timestamp/mnist.py).
```

Comment:
Is it checked in? 
Is earlier one required? Should we keep this as the default and rename to mxnet-mnist-example?

Reply:
The PR https://github.com/kubeflow/katib/pull/970 is required for this.
We can keep it as default mxnet mnist example.
In that case, I will rename mxnet-mnist-timestamp to mxnet-mnist-example and we will collect metrics time in this example also. What do you think about it @johnugeorge ?
/cc @hougangliu @gaocegege

Reply:
agree. thanks